src/app/forms/components/defect/defect.component.ts

```diff
@@ -1,21 +1,33 @@
-import { Component, Input } from '@angular/core';
-import { FormNode } from '@forms/services/dynamic-form.types';
+import { Component, EventEmitter, Input, Output, ViewChild } from '@angular/core';
+import { CustomFormArray, CustomFormGroup, FormNode } from '@forms/services/dynamic-form.types';
 import { DefectTpl } from '@forms/templates/general/defect.template';
 import { Defect } from '@models/defect';
 import { DefectAdditionalInformationLocation } from '@models/defectAdditionalInformationLocation';
+import { DefaultNullOrEmpty } from '@shared/pipes/default-null-or-empty/default-null-or-empty.pipe';
+import { DynamicFormGroupComponent } from '../dynamic-form-group/dynamic-form-group.component';
 
 @Component({
-  selector: 'app-defect',
+  selector: 'app-defect[defect]',
```

Comment:
What does this do?

Reply:
It enforces you to always provide the `defect` input when using `<app-defect>`.